config/webpack/demo/webpack.config.hot.js

```diff
@@ -1,6 +1,7 @@
 "use strict";
 
-var _ = require("lodash"); // devDependency
+var devRequire = require("builder-react-component-dev/require");
```

Comment:
would it make more sense to call this `builderDevRequire` or `builderRequire`? I know it's longer, but it might be easier to infer what's happening.